data/json/monsters/insect_spider.json

```
@@ -589,7 +589,7 @@
     "dissect": "dissect_insect_sample_single",
     "anger_triggers": [ "PLAYER_CLOSE" ],
     "fear_triggers": [ "HURT" ],
-    "upgrades": { "age_grow": 30, "into_group": "GROUP_CENTIPEDE_ADULT" }
+    "upgrades": { "half_life": 30, "into_group": "GROUP_CENTIPEDE_ADULT" }
```

```suggestion
    "upgrades": { "age_grow": 30, "into_group": "GROUP_CENTIPEDE_ADULT" }
```
Centipedes do a bit of dimorphism thing going where they flip between non-reproducing and reproducing morphs, with a low chance to take the fun exit of (sterile) armored centipedes. Using evolution would stop that, since sooner or later they get painted as "non-upgrading".